webapp/lib/controller.dart

```diff
@@ -90,6 +99,23 @@ void initUI() {
   );
 }
 
+void checkNeedsReplyMetricsPipelineDataFlow() {
+  var sortedNeedsReplyDataList = List.from(needsReplyDataList);
+  sortedNeedsReplyDataList.sort((d1, d2) => d1.datetime.compareTo(d2.datetime));
+  var lastNeedsReplyEntry = sortedNeedsReplyDataList.where((entry)=>
+      entry.project == view.contentView.projectSelectorView.selectedProject).last;
+
+  var now = new DateTime.now();
+  var lastUpdateTimeDiff =  now.difference(lastNeedsReplyEntry.datetime).inHours;
+
+  if (lastUpdateTimeDiff > 1) {
```

Comment:
shouldn't this be `lastUpdateTimeDiffInHours >= 1`?

Reply:
Yes